src/debugAdapter/goDebug.ts

```diff
@@ -321,6 +356,18 @@ class GoDebugSession extends DebugSession {
 		this.sendResponse(response);
 		log('ExceptionBreakPointsResponse');
 	}
+    
+    protected toDebuggerPath(path): string {
+        if (this.delve.remotePath.length === 0)
+            return this.convertClientPathToDebugger(path);
+        return path.replace(this.delve.program, this.delve.remotePath);
```

Comment:
Updated to handle case where local and remote paths have different path separators.